ciso/Dockerfile

```diff
@@ -1,10 +1,11 @@
-FROM python:3.11.10-slim
+FROM python:3.11-slim
```

Comment:
This should probably be a RedHat image. There's an available one for 3.11 here: https://catalog.redhat.com/software/containers/ubi9/python-311/63f764b03f0b02a2e2d63fff?gs=&q=python%203.11&container-tabs=overview

We should also have this file watched by `Dependabot` so that updates are done automatically.